.github/workflows/azure-static-webapp.yml

```
@@ -45,7 +45,7 @@ jobs:
 
       - run: |
           cd Uno.Gallery/Uno.Gallery.Wasm
-          dotnet build -c Release "/p:PackageVersion=${{ steps.gitversion.outputs.fullSemVer }}"
+          dotnet msbuild -m:1 -r -p:Configuration=Release "-p:PackageVersion=${{ steps.gitversion.outputs.fullSemVer }}"
```

No change should be needed here.

```suggestion
          dotnet build -c Release "/p:PackageVersion=${{ steps.gitversion.outputs.fullSemVer }}"
```

The font issue happens in CI because it runs on Ubuntu where file paths are case sensitive. This is an issue that was fixed in https://github.com/unoplatform/uno.fonts/pull/31.

A quick workaround is to use Windows in this GitHub Actions, but a better fix is to use the newer package so that everything works if someone is building locally on Linux.

Also, all package dependencies should be updated to roboto 2.2.2